README.md

```diff
@@ -12,206 +12,614 @@ A command-line tool for managing TEE deployments on Phala Network, from local de
+
+**Options:**
+- `-i, --image <image>`: Image name (required)
+- `-t, --tag <tag>`: Image tag (required)
+- `-u, --username <username>`: Docker Hub username
+- `-e, --env-file <envFile>`: Path to environment file
+- `-v, --version <version>`: Template version to use (basic, eliza-v1, eliza-v2)
+
+**Example:**
+```bash
+phala docker build-compose --image my-tee-app --tag v1.0.0 --env-file ./.env
+```
+
+#### Run Local Docker Compose
+
+```bash
+phala docker run-local [options]
+```
+
+Run a Docker Compose file locally for testing.
+
+**Options:**
+- `-c, --compose <compose>`: Path to Docker Compose file
+- `-e, --env-file <envFile>`: Path to environment file
+
+**Example:**
+```bash
+phala docker run-local --compose ./tee-compose.yaml --env-file ./.env
+```
+
+### TEE Simulator Commands
+
+Commands for managing the local TEE simulator for development and testing.
+
+#### Start Simulator
+
+```bash
+phala simulator start [options]
+```
+
+Start the TEE simulator locally for development and testing.
+
+**Options:**
+- `-i, --image <image>`: Simulator image (defaults to 'phalanetwork/phala-pruntime:latest')
+
+**Example:**
+```bash
+phala simulator start
+```
+
+#### Stop Simulator
+
+```bash
+phala simulator stop
+```
+
+Stop the running TEE simulator.
+
+**Example:**
+```bash
+phala simulator stop
+```
+
+### Configuration Commands
+
+Commands for managing CLI configuration settings.
+
+#### Get Configuration Value
+
+```bash
+phala config get <key>
+```
+
+Get a specific configuration value.
+
+**Arguments:**
+- `key`: Configuration key to retrieve
+
+**Example:**
+```bash
+phala config get apiUrl
+```
+
+#### Set Configuration Value
+
+```bash
+phala config set <key> <value>
+```
+
+Set a configuration value.
+
+**Arguments:**
+- `key`: Configuration key to set
+- `value`: Value to set (can be a string, number, boolean, or JSON)
+
+**Example:**
+```bash
+phala config set defaultVcpu 2
+phala config set apiUrl "https://custom-api.phala.cloud"
+phala config set debug true
+phala config set customConfig '{"key": "value", "nested": {"array": [1, 2, 3]}}'
+```
+
+#### List Configuration Values
+
+```bash
+phala config list [options]
+```
+
+List all configuration values.
+
+**Options:**
+- `-j, --json`: Output in JSON format
+
+**Example:**
+```bash
+phala config list
+phala config list --json
+```
+
+### Cloud Virtual Machine (CVM) Commands
+
+Commands for managing Cloud Virtual Machines (CVMs) on Phala Cloud.
+
+#### List CVMs
+
+```bash
+phala cvms list [options]
+```
+
+List all CVMs associated with your account.
+
+**Options:**
+- `-j, --json`: Output in JSON format
+
+**Example:**
+```bash
+phala cvms list
+```
+
+#### Get CVM Details
+
+```bash
+phala cvms get [options] <app-id>
+```
+
+Get detailed information about a specific CVM.
+
+**Arguments:**
+- `app-id`: App ID of the CVM
+
+**Options:**
+- `-j, --json`: Output in JSON format
+
+**Example:**
+```bash
+phala cvms get app_123456
+```
+
+#### Create CVM
```

Comment:
I feel like CVM create/update should have higher priority than the status check above (like list CVMs, teepod, config etc.) thus should be put in front of them

Reply:
But also make sense if you want to make sure first know the status of the environment

Reply:
done